#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_relocate-datadir.dpatch by Jan-Pascal van Best <janpascal@vanbest.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Move the datadir containing the lucene indexes and other stuff
## DP: to /var/lib/solr/data

@DPATCH@
diff -urNad trunk~/example/solr/conf/solrconfig.xml trunk/example/solr/conf/solrconfig.xml
--- trunk~/example/solr/conf/solrconfig.xml	2006-12-16 22:34:35.000000000 +0100
+++ trunk/example/solr/conf/solrconfig.xml	2007-05-31 14:45:25.000000000 +0200
@@ -29,9 +29,8 @@
   <!-- Used to specify an alternate directory to hold all index data
        other than the default ./data under the Solr home.
        If replication is in use, this should match the replication configuration. -->
-  <!--
-  <dataDir>./solr/data</dataDir>
-  -->
+
+  <dataDir>/var/lib/solr/data</dataDir>
 
   <indexDefaults>
    <!-- Values here affect all index writers and act as a default unless overridden. -->
